lib/dns.js

```diff
@@ -278,7 +313,31 @@ exports.getServers = function() {
 };
 
 
-exports.setServers = function(servers) {
+exports.setServers = function setServers(servers) {
```

Comment:
this is no longer guaranteed to `throw` synchronously if there is a problem setting the servers list; perhaps this is a breaking change that makes this PR unworkable?